src/Controls/src/Core/Border.cs

```diff
@@ -312,5 +314,13 @@ void OnStrokeDashArrayChanged(object? sender, NotifyCollectionChangedEventArgs e
 		{
 			Handler?.UpdateValue(nameof(IBorderStroke.StrokeDashPattern));
 		}
+
+		void UpdateStrokeShape()
```

Comment:
When using a Shape in the `StrokeShape` property, we didn't have the `StrokeTickness` mapped from the Border to the Shape.